src/components/title-meta.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import DocumentMeta from "react-document-meta";
+
+class TitleMeta extends React.Component {
```

Comment:
it was in the component-playground-docs repo so I just copied it over! but glad it's a good addition Hooray